docs/hugo/content/guide/metrics.md

```diff
@@ -29,8 +30,71 @@ By default, metrics for ASOv2 are turned on and can be toggled by the following
       containers:
        - args:
          - --metrics-addr=0.0.0.0:8080 (default)    
+         - --secure-metrics=true/false (default: true)
     ```
-   
+
+## Scraping Metrics Securely via HTTPs using RBAC
+
+A ServiceAccount token is required to scrape metrics securely. The corresponding ServiceAccount needs permissions on the "/metrics" and "debug/pprof" paths. 
+This can be achieved e.g. by following the [Kubernetes documentation](https://kubernetes.io/docs/concepts/cluster-administration/system-metrics/).
+
+- Use the settings below in your deployment: 
+
+    ### ASOv2 Helm Chart
+        ```
+        --set metrics.enable=true
+        --set metrics.secure-metrics=true
+        --set metrics.address=0.0.0.0:8443
+        ```
+    
+    ### Deployment YAML
+        ```
+        spec:
+          containers:
+           - args:
+             - --metrics-addr=0.0.0.0:8443  
+             - --secure-metrics=true 
+        ```
+
+- Deploy the following RBAC configuration:
+    ```
+    cat << EOT | kubectl apply -f -
+    apiVersion: rbac.authorization.k8s.io/v1
+    kind: ClusterRole
+    metadata:
+    name: default-metrics
+    rules:
+    - nonResourceURLs:
+        - "/metrics"
+        - "/debug/pprof/*"
+          verbs:
+        - get
+    ---
+    apiVersion: rbac.authorization.k8s.io/v1
+    kind: ClusterRoleBinding
+    metadata:
+    name: default-metrics
+    roleRef:
+      apiGroup: rbac.authorization.k8s.io
+      kind: ClusterRole
+      name: default-metrics
+    subjects:
+    - kind: ServiceAccount
+      name: default
+      namespace: default
+      EOT
+    ```
+- Open a port-forward
```

Comment:
These last two steps are only required for local testing, not actually for something like Prom scraping. Might be worth calling that out somehow?

maybe do something like:
```
- Test locally:
    -  Open a port forward: `kubectl port-forward deployments/azureserviceoperator-controller-manager -n azureserviceoperator-system 8443`
    - Create a ServiceAccount token and scrape metrics:
    ```
    TOKEN=$(kubectl create token default)
    curl https://localhost:8443/metrics --header "Authorization: Bearer $TOKEN" -k
    ```
```